plugins/must-gather/README.md

```
@@ -253,6 +253,34 @@ Active alerts: 2 total (0 pending, 2 firing)
 
 ### Slash Commands
 
+#### `/must-gather:generate [image] [--dest-dir path]`
+Generates must-gather diagnostic data from an OpenShift cluster.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)